.eslintrc.js

```diff
@@ -112,7 +112,7 @@ module.exports = {
     'space-in-parens': [ERROR, 'never'],
     'space-unary-ops': [ERROR, {words: true, nonwords: false}],
     'spaced-comment': [ERROR, 'always', {exceptions: ['*'], markers: ['/']}],
-    'max-depth': [WARN, 4],
+    'max-depth': [WARN, 5],
```

Comment:
bumped this up since there is some deep logic in TableCollection

Reply:
hmm.... i don't like bumping this up, it makes it easier to write that kind of code, how many problem spots are there? can we comment them? or can we pull out any of it to reduce the depth?

Reply:
luckily there are only two instances I believe, I'll suppress the warning for them for now